lib/Doctrine/DBAL/Driver/Mysqli/MysqliStatement.php

```diff
@@ -228,6 +235,59 @@ public function execute($params = null)
         return true;
     }
 
+    /**
+     * Split $this->_bindedValues into those values that need to be sent using mysqli::send_long_data()
+     * and those that can be bound the usual way.
+     *
+     * @return array
+     */
+    private function separateLongData()
+    {
+        $streams = $otherData = [];
+
+        foreach ($this->_bindedValues as $parameter => $value) {
+            if ($this->types[$parameter - 1] !== static::$_paramTypeMap[ParameterType::LARGE_OBJECT]) {
```

Comment:
AFAIK, types may be omitted and default to `STRING`. We probably should check if the value is set.